pkg/queue/schedulingqueue/priority_scheduling_queue.go

```diff
@@ -79,6 +81,14 @@ func (p *PrioritySchedulingQueue) Run() {
 	go wait.Until(p.flushBackoffQCompleted, 1.0*time.Second, p.stop)
 }
 
+func (p *PrioritySchedulingQueue) GetRunStatus() bool {
+	return p.run
```

Comment:
Will there be a race condition case here?

Reply:
Similar to SetRunStatus，Lock the outer layer of the call chain